github/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	Organization string
 	BaseURL      string
 	Insecure     bool
+	Serial       bool
```

Comment:
`Serial` is a pretty generic term, I would ideally opt for something more descriptive.

Reply:
**SerialRequests** maybe?
I'm open to suggestions, naming is always hard... 😞